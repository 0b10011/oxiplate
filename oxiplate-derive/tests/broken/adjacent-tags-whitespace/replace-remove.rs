use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline(r#"{{ "a" _}} {{- "b" }}"#)]
struct Data {}

fn main() {
    let data = Data {};

    print!("{}", data);
}
