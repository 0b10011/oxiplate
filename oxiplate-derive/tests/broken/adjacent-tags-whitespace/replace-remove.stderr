error: Mismatched whitespace adjustment characters
 --> tests/broken/adjacent-tags-whitespace/replace-remove.rs:4:31
  |
4 | #[oxiplate_inline = r#"{{ "a" _}} {{- "b" }}"#]
  |                               ^^^^^^^
